hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -112,6 +111,10 @@ private void processEvent(int cfIndex, byte[] keyBytes, byte[]
       final Object key = cf.getKeyCodec().fromPersistedFormat(keyBytes);
       builder.setKey(key);
 
+      // Initialize table-specific event map if it does not exist
+      omdbLatestUpdateEvents.putIfAbsent(tableName, new HashMap<>());
```

Comment:
Could you add a unit test for this to show that this patch fixes the ClassCastException?

Reply:
@sadanand48 Done!